recipe/emailpassword/recipe.go

```diff
@@ -66,6 +69,12 @@ func MakeRecipe(recipeId string, appInfo supertokens.NormalisedAppinfo, config *
 		if emailVerificationRecipe != nil {
 			emailVerificationRecipe.AddGetEmailForUserIdFunc(r.getEmailForUserId)
 		}
+
+		mtRecipe := multitenancy.GetRecipeInstance()
```

Comment:
email password recipe has no tenant id. So no need to add this at all.

Reply:
This is needed to throw the UNKNOWN_USER_ID correctly